lexicons/tools/ozone/verification/revokeVerifications.json

```diff
@@ -0,0 +1,59 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.verification.revokeVerifications",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Revoke previously granted verifications in batches of up to 100.",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["uris"],
+          "properties": {
+            "uris": {
+              "type": "array",
+              "description": "Array of verification record uris to revoke",
+              "maxLength": 100,
+              "items": {
+                "type": "string",
+                "description": "The AT-URI of the verification record to revoke.",
+                "format": "at-uri"
+              }
+            },
+            "revokeReason": {
+              "type": "string",
+              "description": "Reason for revoking the verification. This is optional and can be omitted if not needed.",
+              "maxLength": 1000
+            }
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["revokedVerifications", "failedRevocations"],
+          "properties": {
+            "revokedVerifications": {
+              "type": "array",
+              "description": "List of verification uris successfully revoked",
+              "items": {
+                "type": "string",
+                "format": "at-uri"
+              }
+            },
+            "failedRevocations": {
+              "type": "array",
+              "description": "List of verification uris that couldn't be revoked, including failure reasons",
+              "items": {
+                "type": "string",
+                "format": "at-uri"
+              }
+            }
```

Comment:
Should this have room for a reason for the failure, similar to `#grantError` in the `grantVerifications` endpoint?

Reply:
yep yep!